align/main.py

```diff
@@ -220,7 +223,28 @@ def extract_netlist_files(netlist_dir,netlist_file):
 
     return netlist_files[0]
 
-def schematic2layout(netlist_dir, pdk_dir, netlist_file=None, subckt=None, working_dir=None, flatten=False, nvariants=1, effort=0, extract=False, log_level=None, verbosity=None, generate=False, regression=False, uniform_height=False, PDN_mode=False, flow_start=None, flow_stop=None, router_mode='top_down', gui=False, skipGDS=False, lambda_coeff=1.0, reference_placement_verilog_json=None, nroutings=1):
+
+def start_viewer(working_dir, json_file):
+    viewer_dir = pathlib.Path(__file__).resolve().parent.parent / 'Viewer'
+    shutil.copytree(viewer_dir, working_dir/'Viewer')
+    shutil.copy(json_file, working_dir/'Viewer'/'INPUT')
+
+    stderr = sys.stderr
+    Handler = functools.partial(http.server.SimpleHTTPRequestHandler, directory=str(working_dir/'Viewer'))
+    with socketserver.TCPServer(('localhost', 0), Handler) as httpd:
+        logger.info(f'Please visit localhost:{httpd.server_address[1]} in your browser')
```

Comment:
You should include the `?design=telescopic_ota_0` part of the url as well in this comment string.
Then you just need to cut and paste.